yellowbrick/features/importances.py

```diff
@@ -168,7 +171,12 @@ def draw(self, **kwargs):
         pos = np.arange(self.features_.shape[0]) + 0.5
 
         # Plot the bar chart
-        self.ax.barh(pos, self.feature_importances_, align='center')
+        #raise Exception(self.feature_importances_)
```

Comment:
Thank you so much for working with on during sprints! I really enjoyed meeting you! One item to fix here, can you remove these comments?